hostscripts/soc-ci/soc-ci

```diff
@@ -250,6 +290,14 @@ def parse_args():
     parser_worker_pool_release.add_argument('slot', type=int)
     parser_worker_pool_release.set_defaults(func=ci_worker_pool_release)
 
+    # workers pool list - list *all* workers pool
+    parser_workers_pool_list = subparsers.add_parser(
+        'workers-pool-list', help='List CI workers pool for all workers')
+    parser_workers_pool_list.add_argument('--my-reservations',
+                                          action='store_true',
```

Comment:
anyway - I changed it so it is now showing only the own reservations. There is now also an ```--all``` switch to show all .